server/vulnerabilities/msrc/xml/vulnerability.go

```diff
@@ -0,0 +1,68 @@
+package xml
+
+import (
+	"fmt"
+	"strings"
+	"time"
+)
+
+// XML elements related to the 'vuln' namespace used to describe vulnerabilities and their remediations.
```

Comment:
same here, if we could document the structs so people like me that are not super familiar with vulnerabilities can catch up more quickly